src/components/Search/SearchFiltersParticipantsSelector.tsx

```diff
@@ -0,0 +1,223 @@
+import React, {useCallback, useEffect, useMemo, useState} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import Button from '@components/Button';
+import {usePersonalDetails} from '@components/OnyxProvider';
+import {useOptionsList} from '@components/OptionListContextProvider';
+import SelectionList from '@components/SelectionList';
+import InviteMemberListItem from '@components/SelectionList/InviteMemberListItem';
+import useDebouncedState from '@hooks/useDebouncedState';
+import useLocalize from '@hooks/useLocalize';
+import * as DeviceCapabilities from '@libs/DeviceCapabilities';
+import * as OptionsListUtils from '@libs/OptionsListUtils';
+import type {Option} from '@libs/OptionsListUtils';
+import type {OptionData} from '@libs/ReportUtils';
+import Navigation from '@navigation/Navigation';
+import * as Report from '@userActions/Report';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+
+const defaultListOptions = {
+    userToInvite: null,
+    recentReports: [],
+    personalDetails: [],
+    currentUserOption: null,
+    headerMessage: '',
+    categoryOptions: [],
+    tagOptions: [],
+    taxRatesOptions: [],
+};
+
+function getSelectedOptionData(option: Option): OptionData {
+    return {...option, selected: true, reportID: option.reportID ?? '-1'};
+}
+
+type SearchFiltersParticipantsSelectorProps = {
+    initialAccountIDs: string[];
+    onFiltersUpdate: (accountIDs: string[]) => void;
+};
+
+function SearchFiltersParticipantsSelector({initialAccountIDs, onFiltersUpdate}: SearchFiltersParticipantsSelectorProps) {
+    const {translate} = useLocalize();
+    const personalDetails = usePersonalDetails();
+    const {options, areOptionsInitialized} = useOptionsList();
+
+    const [betas] = useOnyx(ONYXKEYS.BETAS);
```

Comment:
We can remove this dependency. The only use case is for violations, but we don't show reports in this list.
```suggestion
```